app.py

```diff
@@ -0,0 +1,120 @@
+from flask import Flask, request, jsonify, render_template
+import json
+from jsonschema import validate
+import jsonschema
+import psycopg2
+from psycopg2.extras import RealDictCursor
+
+app = Flask(__name__)
+# run_with_ngrok(app)
+
+POSTGRESQL_URI = 'postgres://axnbpeci:rADWtHAG9X0UDVyrJyvenm8-uX4GjZmR@tyke.db.elephantsql.com/axnbpeci'
+# POSTGRESQL_URI='postgres://postgres:mysecretpassword@postgres/postgres'
+connection = psycopg2.connect(POSTGRESQL_URI)
+
+try:
+    with connection:
+        with connection.cursor() as cursor:
+            cursor.execute("CREATE TABLE users (name VARCHAR(43), surname VARCHAR(43), email VARCHAR(43) UNIQUE NOT NULL, id SERIAL PRIMARY KEY);")
+except psycopg2.errors.DuplicateTable:
+    pass
+
+#Loading schema to a variable
+def load_schema():
+    with open('schema.json') as f:
+        schema = json.load(f)
+    return schema
+
+def add_user_db(data):
+    try:
+        with connection:
+            with connection.cursor() as cursor:
+                sql = 'INSERT INTO users (name, surname, email) VALUES ( %s, %s, %s)'
```

Comment:
This is good for a start. For the next iteration try using an ORM. Something like SQLAlchemy, or similar. Normally we stray away from using directly SQL queries in the code.